index.js

```diff
@@ -1,17 +1,70 @@
 class SortedList {
-  constructor() {}
+  constructor() {
 
-  add(item) {}
+    this.items= []
+    this.length= this.items.length
```

Comment:
Aqui tenias que llamarlos atributos con this y hacer el push del nuevo elemento, recordando tambien el sort()